third_party/openthread/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ set(OT_ANYCAST_LOCATOR ON CACHE STRING "enable anycast locator" FORCE)
 set(OT_BACKBONE_ROUTER ${OTBR_BACKBONE_ROUTER} CACHE STRING "Enable Backbone Router feature in OpenThread" FORCE)
 set(OT_BACKBONE_ROUTER_DUA_NDPROXYING ${OTBR_DUA_ROUTING} CACHE STRING "Configure DUA ND Proxy feature in OpenThread" FORCE)
 set(OT_BORDER_AGENT ON CACHE STRING "enable border agent" FORCE)
+set(OT_BORDER_AGENT_EPSKC ON CACHE STRING "enable border agent ephemeral PSKc" FORCE)
```

Comment:
Not sure if we want to change the source files in `third_party` folder (from a PR in `ot-br-posix`).
I think it should be upstreamed from OT repo?

Reply:
This CMake file lives in `ot-br-posix`. The `openthread` submodule actually lives in `third_party/openthread/repo`. :)

Reply:
I see. Got it. Thanks @jwhui